Source/WebKit/UIProcess/Automation/mac/WebAutomationSessionMac.mm

```diff
@@ -85,6 +85,17 @@
         page.makeFirstResponder();
 
         markEventAsSynthesizedForAutomation(event);
+
+        // NSEventTypeMouseMoved events are not forwarded from the WKWebView to the underlying view implementation,
```

Comment:
They would have been forwarded to the underlying view as a result of the event dispatch, so it would be more correct to simulate this afterwards. Good catch!